elliottlib/template.py

```diff
@@ -0,0 +1,20 @@
+from __future__ import absolute_import, print_function, unicode_literals
+from mako.template import Template
+
+
+def render_topic(template, impact="Important"):
+    t = Template(template)
+    return t.render(impact=impact)
+
+
+def render(template, MAJOR, MINOR):
```

Comment:
I think this function should take `kwargs` rather than hardcoded `MAJOR` `MINOR` arguments.

Reply:
Fixed, please review again.